gpiozero/devices.py

```diff
@@ -34,20 +35,48 @@
 # available, however, we fall back to a pure Python implementation which
 # supports platforms like PyPy
 from .pins import _pins_shutdown
+
+pin_factory = None
+env_default = os.getenv('GPIOZERO_PIN_FACTORY', None)
+
 try:
-    from .pins.rpigpio import RPiGPIOPin
-    pin_factory = RPiGPIOPin
+  if env_default is not None:
```

Comment:
Can you please fix the indents to match the rest of the code? (i.e. 4 spaces per indent, rather than the two you've used here)